package.json

```diff
@@ -25,29 +25,40 @@
     "@wordpress/blocks": "6.21.0",
     "@wordpress/components": "9.4.1",
     "@wordpress/compose": "3.25.0",
+    "@wordpress/core-data": "2.26.0",
     "@wordpress/data": "4.22.3",
     "@wordpress/data-controls": "1.16.3",
     "@wordpress/dom-ready": "2.11.0",
     "@wordpress/edit-post": "3.27.0",
+    "@wordpress/editor": "9.26.0",
     "@wordpress/element": "2.16.0",
+    "@wordpress/escape-html": "1.9.0",
+    "@wordpress/hooks": "2.9.0",
     "@wordpress/i18n": "3.14.0",
     "@wordpress/icons": "2.10.0",
     "@wordpress/keycodes": "2.19.2",
+    "@wordpress/plugins": "2.25.0",
+    "@wordpress/token-list": "1.12.0",
     "@wordpress/url": "2.17.0",
     "classnames": "2.3.1",
     "interpolate-components": "1.1.1",
+    "jquery": "3.5.1",
     "lodash": "4.17.21",
+    "react": "16.11.0",
     "react-animate-height": "2.0.23",
-    "select2": "4.0.13"
+    "select2": "4.0.13",
```

Comment:
Select2 apparently is in a weird situation where it's installed by npm but apparently the copy stored in `node_modules` isn't used. 

Instead, we use the copy stored in `assets/vendor/select2`, for some reason.

We might want to change that in the future (?), but for now, I just ignored that situation.

Reply:
I think we could remove that dependency since we don't use that from the package (and probably we'll just keep using that in the legacy files until we refactor that to something else).

Reply:
Okay, good point. Removed select2 and documented the decision here in the commit 5e2923a6eb658ddb677494c59896aee42813f83b.